### PR TITLE
Us7.1 timeline nav

### PR DESCRIPTION
**Pull request by ktpa**
Let's do this the right way this time.

### CODE REVIEW

File: frontend/src/components/G6Graph.svelte [hunk 1]
```diff
@@ -24,10 +24,10 @@
     export const resizeGraph = () => {
         if (graph && container) {
             const width = Number(
-                window.getComputedStyle(container).width.replace("px", "")
+                window.innerWidth
```
Comment (kam56) [inline frontend/src/components/G6Graph.svelte:27]: would be great if you could add a comment; I do not see any specific change made by this line  :)

**Review by kam56 — APPROVED** [review]
I tested it locally, the graph moves by using the arrow keys. Well done!

Next step: it is hidden functionality atm, but yeah we plan to implement the guidance for it as well.

File: frontend/src/components/G6Graph.svelte (hunk 1 above)
Comment (ktpa) [reply frontend/src/components/G6Graph.svelte:27]: This solves a bug which the getComputedStyle caused.

With getComputedStyle, the window is being computed and below 1080 px wide, it direclty jumped to a set 760 px wide for responsiveness with certain screens. It also had another jump from 760 px to 540 px when the viewport was <760px wide. 

window.innerWidth gives the current width of your viewport regardless of its size, so it allows size 1080 px, also allows 1079 px, 1078 px, ...

The graph is therefore never cut off in certain width intervals with this change. If you want to try it, test the inspect function and play with the viewport width.

Comment (kam56) [reply frontend/src/components/G6Graph.svelte:27]: Well, my first clue was that it removes the right margin created by getComputedStyle. But it was there. And my bad, I hadn't pulled that change. 
It works. Simple solution!

**Comment by ktpa** [comment]
Our WoW doesn't permit merging into master before develop, so closing this and reopening it.

**Pull request closed without merging**